ts/test-electron/linkPreviews/linkPreviewFetch_test.ts

```diff
@@ -122,6 +122,68 @@ describe('link preview fetching', () => {
       );
     });
 
+    const orderedImageHrefSources = [
```

Comment:
Because this is only used in the test below, would you mind moving the variable declaration there?

Reply:
Ah yeah good call, I'm not sure why I put it outside. Done.